services/postprocessing/pkg/config/config.go

```diff
@@ -35,6 +35,7 @@ type Postprocessing struct {
 
 // Events combines the configuration options for the event bus.
 type Events struct {
+	Workers  int    `yaml:"workers" env:"POSTPROCESSING_EVENTS_WORKERS" desc:"The number of concurrent go routines that fetch events from the event queue." introductionVersion:"%%NEXT%%"`
```

Comment:
introductionVersion should be `6.6` or `7.0`. NEXT is only for deprecations as we don't when they will be removed.